src/buckets/pagination/BucketList.tsx

```diff
@@ -45,6 +45,18 @@ class BucketList
   public rowsPerPage: number = 10
   public totalPages: number
 
+  public componentDidMount() {
+    const params = new URLSearchParams(window.location.search)
+    const urlPageNumber = parseInt(params.get('page'), 10)
+
+    const passedInPageIsValid =
+      urlPageNumber && urlPageNumber <= this.totalPages && urlPageNumber >= 0
```

Comment:
I think we want to change `urlPageNumber >=0` to `urlPageNumber > 0` - I think when I first coded this up, I wasn't offsetting the 0 index to 1. Can you play with this and see if 0 or 1 makes a little more sense?

Reply:
Just changed and tested, it works!